### DISCUSSION TITLE
Is there a concrete Road Map of Trame project?

### DISCUSSION THREAD

**Discussion by jealtw** [Kitware/trame#546] [Trame/VTK] [answered]
Hi, I have recently encounter trame project and amazed by its capability, thanks for your great work, it is really amazing!
But I have a question, I want to know that if  there is a concrete Road Map of Trame project? In other words, my question is if trame project is a long term support project so that we can adopt it in our future projects without worry about the sudden closing of this project due to some reason.

**Comment by jourdain** [comment, marked as answer]
One thing that I can say for sure is that trame is not going away. We have way to many projects relying on it at Kitware and in our customer base. 

Regarding the roadmap, I have one, but it is internal. The reason why it is internal, is because as a Open Source company, development is driven by projects and funding and it is hard or almost impossible to provide a timeline for any feature within "my wish list".

What I can say, is that the next big feature is Local rendering using vtk-wasm when using VTK (9.4)/ParaView (5.13/5.14?).

**Reply by jealtw** [reply]
OK, Got it. Thanks for your information.
